### PR TITLE
Fix tests which assume `virtualenv` is available.

### PR DESCRIPTION
**Pull request by mattclay**
##### SUMMARY

Fix tests which assume `virtualenv` is available.

##### ISSUE TYPE

Bugfix Pull Request

##### COMPONENT NAME

integration tests

### CODE REVIEW

File: test/integration/targets/filters/runme.sh [hunk 1]
```diff
@@ -1,10 +1,9 @@
 #!/usr/bin/env bash
 
-set -ex
-virtualenv --system-site-packages --python "${ANSIBLE_TEST_PYTHON_INTERPRETER}" "${OUTPUT_DIR}/venv"
-source "${OUTPUT_DIR}/venv/bin/activate"
 set -eux
```
Comment (webknjaz) [inline test/integration/targets/filters/runme.sh:3]: This is not really related to virtualenv change. But I think we should promote safer mode in scripts:
```suggestion
 set -euxo pipefail
```

File: test/integration/targets/groupby_filter/runme.sh [hunk 2]
```diff
@@ -1,15 +1,8 @@
 #!/usr/bin/env bash
 
-# We don't set -u here, due to pypa/virtualenv#150
-set -ex
+set -eux
```
Comment (webknjaz) [inline test/integration/targets/groupby_filter/runme.sh:3]:
```suggestion
set -euxo pipefail
```

File: test/integration/targets/template_jinja2_latest/runme.sh [hunk 3]
```diff
@@ -1,15 +1,8 @@
 #!/usr/bin/env bash
 
-# We don't set -u here, due to pypa/virtualenv#150
-set -ex
+set -eux
```
Comment (webknjaz) [inline test/integration/targets/template_jinja2_latest/runme.sh:3]:
```suggestion
set -euxo pipefail
```

File: test/integration/targets/lookups/runme.sh [hunk 4]
```diff
@@ -1,10 +1,9 @@
 #!/usr/bin/env bash
 
-set -ex
-virtualenv --system-site-packages --python "${ANSIBLE_TEST_PYTHON_INTERPRETER}" "${OUTPUT_DIR}/venv"
-source "${OUTPUT_DIR}/venv/bin/activate"
 set -eux
```
Comment (webknjaz) [inline test/integration/targets/lookups/runme.sh:3]:
```suggestion
 set -euxo pipefail
```

File: test/integration/targets/inventory_aws_conformance/runme.sh [hunk 5]
```diff
@@ -1,9 +1,9 @@
 #!/usr/bin/env bash
 
-set -ex
+set -eux
```
Comment (webknjaz) [inline test/integration/targets/inventory_aws_conformance/runme.sh:3]:
```suggestion
set -euxo pipefail
```

File: test/integration/targets/filters/runme.sh [hunk 6]
```diff
@@ -1,10 +1,9 @@
...
 
+source virtualenv.sh
```
Comment (webknjaz) [inline test/integration/targets/filters/runme.sh:5]: What is the current working directory in this case? I'm concerned about the relative path...

Comment (mattclay) [reply test/integration/targets/filters/runme.sh:5]: The script is sourced from `$PATH`, so the current working directory doesn't matter.

File: test/integration/targets/filters/runme.sh (hunk 1 above)
Comment (mattclay) [reply test/integration/targets/filters/runme.sh:3]: That's probably something worth changing across all of the tests at once.

None of these tests use pipes, so there's no need to change it in this PR.

File: test/integration/targets/filters/runme.sh (hunk 6 above)
Comment (webknjaz) [reply test/integration/targets/filters/runme.sh:5]: Ah, ok
